fastlane/lib/fastlane/actions/git_add.rb

```diff
@@ -7,9 +7,9 @@ def self.run(params)
           UI.success("Successfully added from \"#{paths}\" 💾.")
         elsif params[:path]
```

Comment:
Hi @ohayon you're right. The reason of keeping the`elseif` here is because these params are conflicting with each other. I'll remove the conflicting option since we don't need it anymore.